tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -592,6 +595,12 @@ def check_args(kwargs):
             print("--test-groups file %s not found" % kwargs["test_groups_file"])
             sys.exit(1)
 
+    if kwargs["fully_parallel"]:
+        if (kwargs["test_groups_file"] is not None or
+            kwargs["run_by_dir"] is not False):
+            print("Can't pass --test-groups or --run-by-dir together with --fully-parallel")
```

Comment:
Maybe a matter of style, but maybe consolidate with the earlier check to make the option constraints obvious (the earlier one doesn't mention `--fully-parallel`)? Something like:

```
if sum([
    kwargs["test_groups_file"] is not None,
    kwargs["run_by_dir"] is not False,
    kwargs["fully_parallel"],
]) > 1:
    print('Must pass up to one of: --test-groups, --run-by-dir, --fully-parallel')
if kwargs["test_groups_file"] is not None and not os.path.exists(...):
    ...
```

Reply:
Done